drop-in/src/main/java/com/adyen/checkout/dropin/internal/ui/DropInActivity.kt

```diff
@@ -627,7 +625,7 @@ internal class DropInActivity :
         val loadingDialog = getFragmentByTag(LOADING_FRAGMENT_TAG)
         if (showLoading) {
             if (loadingDialog == null && !supportFragmentManager.isDestroyed) {
-                LoadingDialogFragment.newInstance().show(supportFragmentManager, LOADING_FRAGMENT_TAG)
+                LoadingDialogFragment.newInstance().showNow(supportFragmentManager, LOADING_FRAGMENT_TAG)
```

Comment:
Any potential issues from calling `showNow`?

Reply:
Afaik no. `show()` is asynchronous, waiting for the main thread to be ready before the dialog is created and show. This will not guarantee the order of dialogs shown.

`showNow()` immediately creates and shows the fragment, guaranteeing the order in which dialogs are created.